templates/log4j.properties.erb

```diff
@@ -0,0 +1,102 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
could you please convert this into an epp template? We prefer this for new templates since that's the erb successor.

Reply:
Yes, sure. Converted